rewrite-core/src/main/java/org/openrewrite/text/Find.java

```diff
@@ -101,6 +101,9 @@ public String getDescription() {
     public TreeVisitor<?, ExecutionContext> getVisitor() {
 
         TreeVisitor<?, ExecutionContext> visitor = new TreeVisitor<Tree, ExecutionContext>() {
+
+            static final int CONTEXT_SIZE = 50;
```

Comment:
Perhaps this should be configurable? Sometimes you might want long matches

Reply:
I the context size is now optional configurable. The default is `0` leading to just the matched text in the Datatable. 
With `-1` the truncate can be deactivated and the full text is in the Datatable, as on main currently for whoever uses this.